src/agenda/index.js

```diff
@@ -92,7 +93,16 @@ export default class AgendaView extends Component {
     /** Called when the momentum scroll starts for the agenda list. **/
     onMomentumScrollBegin: PropTypes.func,
     /** Called when the momentum scroll stops for the agenda list. **/
-    onMomentumScrollEnd: PropTypes.func
+    onMomentumScrollEnd: PropTypes.func,
+    /** Display static header above the calendar on collapse view. Default = Not Showing */
+    staticHeaderOnCollapseView: PropTypes.shape({
```

Comment:
I wonder if this API is clear enough? 
Maybe we can have renderProp here as well? WDYT?

Reply:
I agree the prop's name is unclear